installer/charts/values.yaml.tpl

```diff
@@ -86,6 +91,14 @@ subscriptions:
   redHatQuay:
     enabled: {{ $quay.Enabled }}
     managed: {{ and $quay.Enabled $quay.Properties.manageSubscription }}
+  redHatOpenShiftDataFoundation:
+    enabled: {{ $odfEnabled }}
+    managed: {{ $odfEnabled }}
+    namespace: {{ $odfNamespace }}
+    channel: stable-4.18
```

Comment:
need to figure out how to make this dynamic depending on current openshift version. Take suggestions in the meantime

Reply:
Have a path forward with some changes to variables we fit to the rendering on values.yaml.tpl, working on this now. If this merges before I'm done will send another PR.